AST.fs

```diff
@@ -240,6 +244,43 @@ module Pretty =
     open Starling.Core.TypeSystem.Pretty
     open Starling.Core.Var.Pretty
 
+    /// <summary>
+    ///     Hidden building-blocks for AST pretty-printers.
+    /// </summary>
+    module private Helpers =
```

Comment:
This is a F# feature I didn't realise existed until recently.  It feels like we should have made more use of this throughout the code!